### DISCUSSION TITLE
Mengatur theme html_output

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#129] [Q&A] [answered]
Hello, 

Ingin tau refrensi dalam membuat theme di html_output agar lebih menarik.

Contohnya seperti:
<img width="1406" alt="image" src="https://user-images.githubusercontent.com/99194718/172850492-f0db4d4e-939d-4790-bb94-63281310d192.png">

**Comment by ysitta** [comment]
Halo @jeremiaww,

Apabila ingin membuat Rmd menjadi lebih menarik, Bapak bisa custom/membuat pengaturan sendiri memanfaatkan pengaturan YAML dengan mengatur **theme** dan **highlight**. Adapun untuk referensinya bisa mengunjungi [https://bookdown.org/yihui/rmarkdown/html-document.html](https://bookdown.org/yihui/rmarkdown/html-document.html).

Selain itu, Bapak juga bisa menggunakan **template yang sudah ada** (contohnya pada gambar yang bapak lampirkan). Silahkan Bapak install library **prettydoc** atau **hrbrthemes**. Kemudian, untuk menggunakannya Bapak bisa pilih `New File - RMarkdown - From Template - Pilih tema yang Bapak inginkan`. 

Semoga membantu ya Pak

**Comment by inytss** [comment, marked as answer]
Sebagai tambahan referensi untuk Bapak mengatur tema dalam hasil knitr html Rmarkdown, Bapak dapat melihat galeri tema yang disediakan pada link berikut https://www.datadreaming.org/post/r-markdown-theme-gallery/. 

Untuk contoh tema yang Bapak lampirkan menggunakan library `rmdformats::readthedown`. Bapak dapat membuatnya dengan menginstall package `rmdformats` lalu membuat file Rmd baru dengan `New File > RMarkdown > From Template > Pilih HTML readthedown template`. 

Apabila jawaban ini membantu, Bapak dapat menekan tombol `Mark as answer` agar saya tahu bahwa jawaban ini membantu.

**Reply by jeremiaww** [reply]
Baik Mba, terima kasih atas informasi dan bantuannya.

Secepatnya akan saya coba untuk fungsi nya.
